README.md

```diff
@@ -11,9 +11,11 @@ into your bash scripts using `source`.
 ## Examples
 
 Once you have `bash-commons` installed (see the [install instructions](#install)), you use `source` to import the
-modules and start calling the functions within them:
+modules and start calling the functions within them. Before you import any modules, make sure you `source` the
+`bootstrap.sh` file which sets some important defaults which encourages good code:
```

Comment:
s/which encourages/which encourage/

Reply:
Isn't `encourages` the correct tense though?

Reply:
Ha, turns out this sentence is ambiguous! If "which" refers to `bootstrap.sh` (singular), then "encourages" is correct; if it refers to "important defaults" (plural), then it should be "encourage." Perhaps change: "to encourage good code."

Reply:
Haha you're right. I do have a track record of writing ambiguous sentences. It's a good thing code doesn't allow for this :p